doc/rst/platforms/cray.rst

```diff
@@ -535,9 +510,7 @@ When hugepages are used with the ugni comm layer, tasking layers
 cannot use guard pages for stack overflow detection.  Qthreads tasking
 can only use guard pages for stack overflow detection, so if ugni
 communications is combined with qthreads tasking, overflow detection is
-turned off completely.  Muxed tasking can use guard pages for stack
-overflow detection, but it can also drop back to synchronous overflow
-detection, as described below, with ``CHPL_COMM=ugni`` and hugepages.
+turned off completely.
```

Comment:
Yeah, okay, I didn't spot that part of it.  It's still a little misleading, I think, but certainly any improvement could wait until #5523.